cypher-manual/extensions.md

```diff
@@ -0,0 +1,50 @@
+---
+id: extension-clauses
+title: Memgraph Cypher extensions 
+sidebar_label: Memgraph Cypher extensions
+---
+
+### Foreach
+
+***foreach*** iterates over a list of elements. Each element is stored inside a variable which can
+optionally be used inside the update clauses. All update clauses are executed per iteration of the list.
+
+```cypher
+  FOREACH ( <variable name> IN <expression> | <update clauses> )
+```
+|         Option          |                                                         Description                                                       |
+| :---------------------: | :-----------------------------------------------------------------------------------------------------------------------: |
+|       variable name     |                                         The variable name that stores each element                                        |
+|        expression       |                                           Any expression that results to a list                                           |
+|      update clauses     |                  One or more cypher update clauses: SET, REMOVE, CREATE, MERGE, DELETE including FOREACH extension        |
+
+It must be noted that if the result &lt;expression&gt; is null, then **foreach** will not fail but rather skip the execution of &lt;update clauses&gt; alltogether.
```

Comment:
```suggestion
It must be noted that if the result `&lt;expression&gt;` is null, then `FOREACH` will not fail but rather skip the execution of `&lt;update clauses&gt;` altogether.
```

Reply:
I was wrong. The back tick and &lt and &gt doesn't work well. So I think just go with \`<expression>\` and \`update clauses\`.